packages/ia-topnav/src/models.ts

```diff
@@ -0,0 +1,114 @@
+export interface IATopNavConfig {
+  /**
+   * Google Analytics event category
+   */
+  eventCategory?: string;
+
+  /**
+   * Copy to display for number of pages archived at the top of the Wayback search form
+   *
+   * ie. "425 billion"
+   */
+  waybackPagesArchived?: string;
```

Comment:
do we have default values for these strings?

Reply:
Not in the interface, but we have a `defaultTopNavConfig` that provides a default value. https://github.com/internetarchive/iaux/pull/946/files#diff-4f0ea7c801725a8cc9680a7d6c698b20fddcc40bea5e6b6150e36512ed554e7aL9